pkg/operator/api/resource/status.go

```diff
@@ -92,25 +92,23 @@ const (
 	StatusParentKilled
 
 	// Data statuses
-	StatusDataRunning
-	StatusDataSucceeded
-	StatusDataFailed
-	StatusDataKilled
+	StatusRunning
+	StatusSucceeded
+	StatusFailed
+	StatusKilled
 
 	// API statuses
-	StatusAPIUpdating
-	StatusAPIReady
-	StatusAPIStopping
-	StatusAPIStopped
-	StatusAPIError
+	StatusUpdating
+	StatusReady
+	StatusStopping
+	StatusStopped
+	StatusError
 
 	// Additional API group statuses (i.e. aggregated API status)
-	StatusAPIGroupPendingUpdate
-	StatusAPIGroupParentFailed
-	StatusAPIGroupParentKilled
-	StatusAPIGroupUpdateSkipped
+	StatusPendingUpdate
+	StatusUpdateSkipped
 
-	StatusDataKilledOOM
+	StatusKilledOOM
```

Comment:
If we're keeping the comments (e.g. `// Data statuses`), then we should move this to the `// Shared statuses` section. If you don't want to change the order, we can remove the comments or add an inline comment for this one (e.g. `// Shared status`)